src/tacacs/0001-Don-t-enable-pam-tacplus-by-default.patch

```diff
@@ -0,0 +1,51 @@
+From 80c1d3c1810bf283bbe12fc927de24e48afc2991 Mon Sep 17 00:00:00 2001
+From: Liuqu <chenchen.qcc@alibaba-inc.com>
```

Comment:
I suggest not to change the way the patch libpam-tacplus, instead, we can disable the pam-tacplus by default in our image. the idea to minimize the change to the original package as little as possible.

Reply:
Add some scripts to disable the pam-tacplus after install libpam-tacplus?